src/main/java/Main.java

```diff
@@ -37,41 +24,82 @@ public static void main(String[] args) {
         manufacturerService.create(manufacturer3);
         manufacturerService.create(manufacturer4);
         manufacturerService.create(manufacturer5);
-        Car car1 = new Car("model1", manufacturer1);
-        Car car2 = new Car("model2", manufacturer2);
-        Car car3 = new Car("model3", manufacturer3);
-        Car car4 = new Car("model4", manufacturer4);
-        Car car5 = new Car("model5", manufacturer5);
-        carService.create(car1);
-        carService.create(car2);
-        carService.create(car3);
-        carService.create(car4);
-        carService.create(car5);
+        manufacturerService.getAll().forEach(System.out::println);
+        Manufacturer changedManufacturer1 = new Manufacturer("blabla", "blabla");
+        changedManufacturer1.setId(5L);
 
+        manufacturerService.update(changedManufacturer1);
+        manufacturerService.delete(1L);
         manufacturerService.get(1L);
-        manufacturerService.delete(2L);
-        manufacturer1.setCountry("nusa").setName("nvehicle");
-        manufacturerService.update(manufacturer1);
-        List<Manufacturer> allManufacturers = manufacturerService.getAll();
-        allManufacturers.forEach(System.out::println);
+        //        manufacturerService.delete(2L);
```

Comment:
we shouldn't leave commented code in the PR. If we don't need it, we should delete it